Source/Tools/LinuxEmulation/LinuxSyscalls/x32/Ioctl/drm.h

```diff
@@ -1115,10 +1115,10 @@ namespace I915 {
     fex_drm_i915_cmdbuffer_t(drm_i915_cmdbuffer_t val)
       : buf {auto_compat_ptr {val.buf}}
       , cliprects {auto_compat_ptr {val.cliprects}} {
-      CPYT(sz);
-      CPYT(DR1);
-      CPYT(DR4);
-      CPYT(num_cliprects);
+      CPYF(sz);
```

Comment:
Personally I find the CPYT/CPYF macros actually make it harder to verify the code is doing the correct thing.

Looks like the true risk here is to have the `val` parameter not be const, so that assigning to its members would be a compile-error. The other case (using CPYF instead of CPYT) would be covered if we used brace-initialization instead of member-wise init.

Not sure it's worth going through the entire header to fix this though. Arguably the framework driving `thunkgen` could automate all of it.

Reply:
Should be solved in a follow-up PR. The idiom is already in place from before this PR, this just extends it to all the structs. With it covering all the structs we will more easily be able to sed replace them, knowing their correct.